client/visualizer/electron-main.js

```diff
@@ -52,6 +52,11 @@ function createWindow () {
   });
 }
 
+// This is needed so that the speedscope iframe on the Profiler tab does not throw cross-origin errors
+// The speedscope iframe loads speedscope using the file:// protocol which considers all URLs to have separate origins
+// The only way to get around this is to tell Chrome to allow it anyways
+electron.app.commandLine.appendSwitch('disable-site-isolation-trials');
```

Comment:
this seems less than ideal but maybe it is inevitable

(just commenting so i remember to go back to this)

Reply:
When this line is removed, the Profiler tab works when the application is first started but breaks when the user hits Ctrl+R. There may be other solutions to this, but it already took me 30 minutes to fix it using this line and I also had a sprint bot to work on :)

Reply:
oh i see, let's just leave this here then